include/systems/lobby_system.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef FILE_LOBBY_SYSTEM_HPP
+#define FILE_LOBBY_SYSTEM_HPP
+
+#include <vector>
+#include <utility>
+
+#include "systems/beast_system.hpp"
+#include "entities/entity_factory.hpp"
+#include "enums/character.hpp"
+
+class LobbySystem : public BeastSystem {
+public:
+    LobbySystem(std::shared_ptr<EntityFactory> ef, std::shared_ptr<EntityManager> em);
+    void update(double deltatime);
+private:
+    std::shared_ptr<EntityFactory> ef;
+    std::shared_ptr<EntityManager> em;
+
+    std::vector<std::pair<Character, bool>> picked_characters = {
```

Comment:
These should be reset after the level has loaded. After one round with four players, I can only choose randoms.

Reply:
Good call, I implemented the reset function in the LobbySystem to fix this